### ISSUE TITLE
Password: TypeError: Cannot read properties of undefined (reading 'inputStyle')

### ISSUE THREAD

**Issue by MAN-Sendance** [primefaces/primeng#15717]
### Describe the bug

I have the following code for adding a password field to my application:

```
<div class="form-control-group">
            <label class="block font-semibold mb-2" for="login-password"
              >Password</label
            >
            <span class="p-input-icon-left inline">
              <mat-icon
                aria-hidden="false"
                aria-label="password icon"
                class="material-icons-outlined -translate-y-50 top-50 absolute ml-2"
                fontIcon="lock"
              />
              <input
                #password="ngModel"
                [(ngModel)]="user.password"
                [attr.aria-invalid]="
                  password.invalid && password.touched ? true : null
                "
                [required]="
                  getConfigValue('forms.validation.password.required')
                "
                [feedback]="false"
                aria-describedby="password-help"
                class="w-full"
                [ngClass]="{
                  'mb-3':
                    password.valid || password.untouched || !password.dirty
                }"
                id="login-password"
                name="login-password"
                pPassword
                placeholder="Password"
                type="password"
              />
            </span>
            <ng-container
              *ngIf="password.invalid && password.touched && password.dirty"
            >
              <div id="password-help" class="text-red-500 mb-3 mt-1">
                <span class="text-sm">{{
                  password.errors?.['required']
                    ? 'Password is required!'
                    : 'Invalid input!'
                }}</span>
              </div>
            </ng-container>
          </div>
```

It had been working perfectly until it suddenly stopped working in v17.16.0

### Environment

don't know what to tell

### Angular version

17.3.4

### PrimeNG version

17.17.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

20.13.1

### Browser(s)

All

### Expected behavior

`pPassword` directive should work again ...

**Comment by mehmetcetin01140** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue clearly?

**Comment by MAN-Sendance** [comment]
It seems to have been unexpectedly fixed in any version `> 17.16.0`.
